# HG changeset patch
# User Rik <rik@octave.org>
# Date 1549762073 28800
#      Sat Feb 09 17:27:53 2019 -0800
# Node ID 4a11a8ae0a04cd952350470880bb54b89cfaa56a
# Parent  f8e7c175022265b6d840e094ed87379a0f2495da
Fix jit BIST test accidentally failing after cc0d942d0e20.

* jit.tst: Remove unnecessary unwind_protect block.

diff --git a/test/jit.tst b/test/jit.tst
--- a/test/jit.tst
+++ b/test/jit.tst
@@ -372,22 +372,20 @@
 %!   i0 = i0 + 1;
 %! endwhile
 %! expected = ones (ndim, ndim, ndim, ndim);
 %! assert (all (m == expected));
 %! assert (result == sum (expected (:)));
 %! assert (jit_failcnt, 0);
 
 %!function test_divide ()
-%! unwind_protect
-%!   for i=1:1e5
-%!     a = 1;
-%!     a / 0;
-%!   endfor
-%! end_unwind_protect
+%! for i=1:1e5
+%!   a = 1;
+%!   a / 0;
+%! endfor
 %!endfunction
 
 # %!testif HAVE_LLVM
 # %! jit_failcnt (0);
 # %! lasterr ("");
 # %! try
 # %!   test_divide ();
 # %! end_try_catch
